i18n/react-intl/en.json

```diff
@@ -71,14 +71,24 @@
   "peopleIntro": "Processing is a community effort led by a small group of volunteers.",
   "booksIntro": "Processing books cover topics from programming basics to visualization. Browse this page to find the right books for you.",
   "readMore": "Read more",
-  "downloadIntro": "Processing is available for Linux, Mac, and Windows. Select your choice to download the software below.",
-  "report": "Report Bugs",
-  "supported": "Troubleshooting",
-  "downloadChanges": "Read about the changes in 4.0. <a href='https://github.com/processing/processing4/blob/master/build/shared/revisions.md'>This list of revisions</a> covers the differences between releases in detail.",
-  "earlierReleases": "Earlier releases have been removed because we can only support the current versions of the software. Read about <a href='https://github.com/processing/processing4/blob/main/build/shared/revisions.md'>revisions between individual releases</a>.",
-  "preReleases": "Pre-Releases",
-  "downloadOutro": "<a href='https://github.com/processing/processing4/wiki/Changes-in-4.0'>Read about changes in 4.0 since 3.x</a>.",
+  "downloadIntro": "With its vibrant community and endless resources, Processing is the perfect free and open-source tool for artists, designers, educators, and anyone looking to explore the creative side of coding. Download Processing and start creating today!",
```

Comment:
"downloadIntro": "Processing is open source and is available for macOS, Windows, and Linux. Projects created with Processing are also cross-platform, and can be used on macOS, Windows, Android, Raspberry Pi, and many other Linux platforms."